projects/static/projects/js/fundings.js

```diff
@@ -0,0 +1,50 @@
+function updateElementIndex(el, ndx) {
+    var id_regex = new RegExp('(form-\\d+)');
+    var replacement = 'form-' + ndx;
+    if ($(el).attr("for")) $(el).attr("for", $(el).attr("for").replace(id_regex, replacement));
+    if (el.id) el.id = el.id.replace(id_regex, replacement);
+    if (el.name) el.name = el.name.replace(id_regex, replacement);
+}
+
+function cloneMore(selector) {
+    var newElement = $(selector).clone(true);
```

Comment:
I was looking at how one is "supposed" to do this kind of thing in Django and [came across something familiar](https://stackoverflow.com/questions/501719/dynamically-adding-a-form-to-a-django-formset) -- I think it is good to include source URLs when code is used like this, just so it's clearer where it came from (and the source may have more details about the reason.)

I wish there was a better solution for this! But, I'm having trouble thinking of one. The fundamental thing this is doing seems correct, cloning the last and then updating all the form refs so that Django does the right thing on the POST.